docs/src/indexing-flow.md

```diff
@@ -79,6 +79,32 @@ add_filter( 'algolia_should_index_user', 'filter_user', 10, 2 );
 In the above example, User with ID 1 would never get indexed. This example is trivial, but at least gives you a quick overview on how to filter your items to index.
 Also note how we return `false` early on if the decision has already been taken to not index the content.
 
+**Another example to exclude posts from the searchable_posts index where the noindex option from the Yoast SEO plugin is set to "noindex"**
+```php
```

Comment:
No sorry, I'd rather make it crystal clear with some specific formatting. Do you mind adding the suggested alert?